subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/serialization/codecs/Codecs.kt

```diff
@@ -191,7 +192,7 @@ class Codecs(
         bind(ProxyCodec)
 
         // Java serialization integration
-        bind(unsupported<Externalizable>())
+        bind(unsupported<Externalizable>(DocumentationSections.notYetImplemented))
```

Comment:
:x: this should link to something like `config_cache:not_yet_implemented:java_serialization` for a new section about not yet implemented Java serialization idioms

Reply:
Done.